src/cli.ts

```diff
@@ -189,13 +198,14 @@ export function ensureGitHubToken(options: CliOptions) {
     try {
       options['github-token'] = fs.readFileSync('github-token', 'utf8').trim();
     } catch (e) {
-      console.error(`
+      const err = new Error(`
```

Comment:
I've never seen this before, is this something you've used before or just because of how main report errors? Either way a comment would be nice explaining why you're removing the stack. (Or if possible, change the error handling to log the entire error object. `console.error(err)` should format both the message and the stack properly.)